index.js

```diff
@@ -6,28 +6,29 @@ const _ = require('lodash'),
   yml = require('js-yaml'),
   { getComponentName } = require('clayutils');
 
-var log = require('./services/log').setup({ file: __filename });
+let log = require('./services/log').setup({ file: __filename });
 
 /**
- * readFile
- *
  * memoizes function that reads and translates YAML file
  *
  * @param {String} filePath
  * @returns {Object} yml file's contents as a Javascript object
+ * @throws {Error}
  */
-const loadYml = _.memoize(function readFile(filePath) {
-  try {
-    const file = files.readFileSync(filePath, 'utf8');
+function loadYml(filePath) {
+  return _.memoize(function () {
```

Comment:
Not sure this should be moved, doesn't this mean that now every time we call this function we are returning a new memoized version of the loadYml routine?

Reply:
I did the change because the linter was yelling about having multiple `const` statements. But now that you mention it, you're right. I'll just declare it as a `let` and should work the same way